deployment/network-operator/values.yaml

```diff
@@ -112,6 +112,7 @@ sriov-network-operator:
 # General Operator related values
 # The operator element allows to deploy network operator from an alternate location
 operator:
+  resources: {}
```

Comment:
Do we need this?

Reply:
This is a placeholder left to remind that operator resources can also be customized